internal/service/cloudbackupsnapshotexportbucket/resource_cloud_backup_snapshot_export_bucket.go

```diff
@@ -72,9 +73,10 @@ func Schema() map[string]*schema.Schema {
 			ForceNew: true,
 		},
 		"tenant_id": {
-			Type:     schema.TypeString,
-			Optional: true,
-			ForceNew: true,
+			Deprecated: fmt.Sprintf(constant.DeprecationParamByVersion, "1.27.0") + " The tenantId of the Cloud Provider Access role (from roleId) is used.",
```

Comment:
related to above comment https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2932/files#r1898457079, can DeprecationParamByVersionWithReplacement be used?

Also, wanted to confirm if 1.27 for deprecation has been discussed? Asking since we have ongoing discussions around deprecations, so given that does DeprecationParamFutureWithReplacement make more sense?

Reply:
I think this needs a discussion.
IMO: This is a "soft" breaking change where the user only needs to remove an attribute.
Maybe add prefix instead: `Safe to remove. The tenantId ....`

I believe the DeprecationParamByVersionWithReplacement text can be missleading. As it says `Please transition to %s`

Reply:
no strong opinions on the exact wording, as long as we can discuss internally and align on an approach

Reply:
Updated in 2474fb9f1044fe5a84eb1bfe4f7c99682e51ab29
Added to tech agenda.